src/platform/silabs/multi-ota/SiWx917/OTAWiFiFirmwareProcessor.cpp

```diff
@@ -63,6 +63,14 @@ CHIP_ERROR OTAWiFiFirmwareProcessor::ProcessInternal(ByteSpan & block)
     }
 
     OTATlvProcessor::vOtaProcessInternalEncryption(byteBlock);
+    if (IsLastBlock() == true)
+    {
+        // Unpad the file if it is the last block
```

Comment:
This documentation should be moved to either the method declaration (header), or the method definition (.cpp). Header is preferable.